config/default.toml

```diff
@@ -0,0 +1,37 @@
+# Valid time zones are listed under the "TZ" column at
+# https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+timezone = "Etc/UTC"
```

Comment:
Nit: Can we keep the current values as the default values? This would be `'America/Chicago'`.